sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/models/Sgv2IndexAddRequest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.restsvc.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.stargate.sgv2.common.cql.builder.CollectionIndexingType;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+public class Sgv2IndexAddRequest {
```

Comment:
I think we'll need to add `@ApiModel("IndexAddRequest") -- or whatever external name we want -- for Swagger to use. Otherwise will use unqualified class name as external name.

Reply:
Is that model name surfaced anywhere? I don't see it in the [generated docs](https://stargate.io/docs/stargate/1.0/attachments/restv2.html#operation/createIndex).

Reply:
/swagger-ui endpoint exposes it under `Models`.